crates/solidity/inputs/language/src/definition.rs

```diff
@@ -132,59 +132,51 @@ codegen_language_macros::compile!(Language(
-                                        fields = (operator = Required(Minus))
-                                    )]
-                                ),
-                                PrecedenceExpression(
-                                    name = VersionComparator,
-                                    operators = [
-                                        PrecedenceOperator(
-                                            model = Prefix,
-                                            fields = (operator = Required(Caret))
-                                        ),
-                                        PrecedenceOperator(
-                                            model = Prefix,
-                                            fields = (operator = Required(Tilde))
-                                        ),
-                                        PrecedenceOperator(
-                                            model = Prefix,
-                                            fields = (operator = Required(Equal))
-                                        ),
-                                        PrecedenceOperator(
-                                            model = Prefix,
-                                            fields = (operator = Required(LessThan))
-                                        ),
-                                        PrecedenceOperator(
-                                            model = Prefix,
-                                            fields = (operator = Required(GreaterThan))
-                                        ),
-                                        PrecedenceOperator(
-                                            model = Prefix,
-                                            fields = (operator = Required(LessThanEqual))
-                                        ),
-                                        PrecedenceOperator(
-                                            model = Prefix,
-                                            fields = (operator = Required(GreaterThanEqual))
-                                        )
-                                    ]
-                                )
-                            ],
-                            primary_expressions = [
-                                PrimaryExpression(reference = VersionSpecifiers),
-                                PrimaryExpression(reference = SingleQuotedVersionLiteral),
-                                PrimaryExpression(reference = DoubleQuotedVersionLiteral)
+                            variants = [
+                                EnumVariant(reference = VersionRange),
+                                EnumVariant(reference = VersionTerm)
+                            ]
+                        ),
+                        Struct(
+                            name = VersionRange,
+                            fields = (
+                                lower = Required(VersionTerm),
```

Comment:
suggestion: using `start`/`operator`/`end` instead of `lower`/`minus`/`upper`, to match the rest of the grammar.

Reply:
Done